include/gatekeeper_lls.h

```diff
@@ -359,6 +361,10 @@ ipv6_addrs_equal(const uint8_t *addr1, const uint8_t *addr2)
 	return (paddr1[0] == paddr2[0]) && (paddr1[1] == paddr2[1]);
 }
 
+/* Submit ARP packets to the LLS block (hardware filtering is not available). */
+int submit_arp(struct rte_mbuf **pkts, unsigned int num_pkts,
```

Comment:
No caller of `submit_arp()` uses its return, and `submit_arp()` already deals with falures. Shouldn't we make the return of `submit_arp()` `void`?